lisk-dex-electron/src/PlaceOrder.jsx

```diff
@@ -318,6 +329,23 @@ export default class PlaceOrder extends React.Component {
     }
   }
 
+  getAssetBalance() {
+    const whole = 10 ** 8;
+    return Math.round((this.props.assetBalance * 100) / whole) / 100;
+  }
+
+  getBaseFee() {
+    const dexConfig = this.props.configuration.markets[this.props.activeMarket].marketOptions;
+    const chains = Object.keys(dexConfig.chains);
+    const firstChain = dexConfig.chains[chains[0]];
+    const secondChain = dexConfig.chains[chains[1]];
+    const baseFeeKey = 'exchangeFeeBase';
+    const keyDescriptor = marketInfoDescriptor[baseFeeKey];
+    const firstChainBaseFee = firstChain[baseFeeKey] / keyDescriptor.div;
+    const secondChainBaseFee = secondChain[baseFeeKey] / keyDescriptor.div;
```

Comment:
The `div` value should come from the `config.json` file because it is configurable https://github.com/Leasehold/ldex-ui/blob/master/lisk-dex-electron/public/config.json#L13

It's important that all the important info about any market comes from only two places:

- The `config.json` file
- The `/status` HTTP API endpoint

Inside `Market.js`, the `div` property for `minOrderAmount`, `minPartialTake`, `exchangeFeeBase` should not be there since this value depends on the market so it cannot be hardcoded.

Reply:
Ohk... actually those values were hardcoded in the code itself, I moved them to market.js

Reply:
Didn't know I can actually get them from config.json, will make then necessary change

Reply:
Can we create a new issue for this one? I think impact will be at several places.